pyproject.toml

```diff
@@ -22,6 +22,85 @@ ignore_errors = true
 fail_under = 90.0
 show_missing = true
 
+[tool.poetry]
+name = "sceptre"
+version = "3.1.0"
+description = "Cloud Provisioning Tool"
+authors = ["Cloudreach <sceptre@cloudreach.com>"]
+maintainers = []
+license = "Apache-2.0"
+readme = "README.md"
+homepage = "https://docs.sceptre-project.org/"
+repository = "https://github.com/Sceptre/sceptre"
+documentation = "https://docs.sceptre-project.org/"
+keywords = ["sceptre", "aws", "cloud", "provisioning", "cloudformation"]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Natural Language :: English",
+  "Environment :: Console",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+]
+packages = [
+  { include = "sceptre/" },
+]
+include = ["CHANGELOG.md", "README.md"]
+
+[tool.poetry.dependencies]
```

Comment:
how does this work with the current set of requirements in `requirements` folder?

Reply:
Those all go away; they're no longer needed since all dependencies/requirements are handled by Poetry.